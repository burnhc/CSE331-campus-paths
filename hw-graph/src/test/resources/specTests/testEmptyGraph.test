# Tests createGraph and that an empty graph is represented properly.

# Create a new graph.
CreateGraph graph1

# Print all the nodes in the graph.
ListNodes graph1