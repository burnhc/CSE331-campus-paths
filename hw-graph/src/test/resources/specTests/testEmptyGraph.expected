# Tests createGraph and that an empty graph is represented properly.

# Create a new graph.
created graph graph1

# Print all the nodes in the graph.
graph1 contains: